src/script/notification/NotificationRepository.ts

```diff
@@ -672,7 +672,7 @@ export class NotificationRepository {
       };
     }
 
-    return () => amplify.publish(WebAppEvents.CONVERSATION.SHOW, conversationEntity || conversationId, {});
+    return () => amplify.publish(WebAppEvents.CONVERSATION.SHOW, conversationEntity || conversationId, {archive: conversationEntity?.archivedState()});
```

Comment:
We can read `archivedState` in `CONVERSATION.SHOW` event listener (`showConversation` method) directly, no need to pass it here.